main/views_test.py

```diff
@@ -61,3 +61,12 @@ def test_cybersource_context(client, user):
         "bootcamp_run_purchased": fake_title,
         "purchase_date_utc": fake_datetime,
     }
+
+
+@pytest.mark.django_db
```

Comment:
That is not required as well.

Reply:
@umarmughal824  The test fails without this annotation. Since client redirection is unsuccessful while ensuring connection because of database access failure giving `Failed: Database access not allowed, use the "django_db" mark` error. 
Can you please suggest another approach that should be followed here?

Reply:
If that error is occurred, keep it intact.